src/app/shared/services/compare.service.ts

```diff
@@ -630,7 +630,7 @@ export class CompareService implements OnDestroy {
     return {
       'color': [current.color[0], compare.color[0], current.color[1], compare.color[1]],
       'textStyle': { ...current.textStyle },
-      'grid': { ...current.grid, top: 32 },
+      'grid': { ...current.grid, top: 32, right: 50, left: 50 },
```

Comment:
@diamond-darrell  - this will affect all our graphs. I'll try to find another solution so let's keep this open for now